code/addons/docs/src/blocks/components/DocsPage.tsx

```diff
@@ -255,6 +255,20 @@ export const DocsContent = styled.div(({ theme }) => {
       color: theme.color.defaultText,
       '& code': code,
     },
+    // Ensure keyboard focus is visible for interactive elements inside docs
+    [toGlobalSelector('a, button, input, textarea, select')]: {
+      '&:focus-visible': {
```

Comment:
I think we'd be better off targeting `toGlobalSelector('*:focus-visible')`. We're missing things like `area`, `video controls`, `summary`.